### PR TITLE
[Frugalware.org] Remove wildcards, rename and reactivate ruleset

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

**Comment by Bisaloo** [comment]
just noticed it's in #3069

File: src/chrome/content/rules/Frugalware.org.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	<securecookie host="^.+\.frugalware\.org$" name=".+" />
+
+	<rule from="^http://lists\.frugalware\.org/"
+		to="https://frugalware.org/mailman/listinfo/" />
+
+	<rule from="^http://packages\.frugalware\.org/"
+		to="https://frugalware.org/packages/" />
+
+	<rule from="^http://planet\.frugalware\.org/"
+		to="https://frugalware.org/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Frugalware.org.xml]: This does not work for deep links.

File: src/chrome/content/rules/Frugalware.org.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		dors.frugalware.org
+		git.frugalware.org
+		lists.frugalware.org
+		packages.frugalware.org
+		planet.frugalware.org
+		wiki.frugalware.org
+
+	Private subdomain:
+		webmail.frugalware.org
+
```
Comment (J0WI) [inline src/chrome/content/rules/Frugalware.org.xml]: There are also some mirrors like `www\d.frugalware.org` or `rsync\d.frugalware.org`

File: src/chrome/content/rules/Frugalware.org.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+		darcs.frugalware.org
+		dors.frugalware.org
+		git.frugalware.org
+		lists.frugalware.org
+		packages.frugalware.org
+		planet.frugalware.org
+		wiki.frugalware.org
+
+	Private subdomain:
+		webmail.frugalware.org
```
Comment (J0WI) [inline src/chrome/content/rules/Frugalware.org.xml]: As long as we can get a HTTP 200 response we should add it anyway. Maybe some people there are using HTTPS Everywhere too ;)

File: src/chrome/content/rules/Frugalware.org.xml (hunk 1 above)
Comment (Bisaloo) [inline src/chrome/content/rules/Frugalware.org.xml]: What do you mean? Would you have a test url to explain?

File: src/chrome/content/rules/Frugalware.org.xml (hunk 3 above)
Comment (Bisaloo) [inline src/chrome/content/rules/Frugalware.org.xml]: The argument against adding private subdomains was that we don't know if the rewrite rule will break what's behind the login page: maybe there is active mixed content.

File: src/chrome/content/rules/Frugalware.org.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/rules/Frugalware.org.xml]: e.g. http://planet.frugalware.org/index.php?post_id=27 will not work with this rule.

File: src/chrome/content/rules/Frugalware.org.xml (hunk 2 above)
Comment (J0WI) [inline src/chrome/content/rules/Frugalware.org.xml]: LGTM, apart from this

Comment (Bisaloo) [inline src/chrome/content/rules/Frugalware.org.xml]: It looks like none of them are working over https, would you accept something like:

```xml
	Non functional over https for various reasons:
		www\d.frugalware.org
		rsync\d.frugalware.org
```

?

Comment (J0WI) [inline src/chrome/content/rules/Frugalware.org.xml]: Yes, that's okay

**Pull request merged**